exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -99,48 +105,44 @@ public void Roll(int roll)
                 Console.WriteLine(players[currentPlayer]
                         + "'s new location is "
                         + places[currentPlayer]);
-                Console.WriteLine("The category is " + CurrentCategory());
+                Console.WriteLine("The category is " + SubjectOfQuestion());
                 AskQuestion();
             }
 
         }
 
         private void AskQuestion()
         {
-            if (CurrentCategory() == "Pop")
+            if (SubjectOfQuestion() == POP_CATEGORY)
             {
                 Console.WriteLine(popQuestions.First());
                 popQuestions.RemoveFirst();
             }
-            if (CurrentCategory() == "Science")
+            if (SubjectOfQuestion() == SCIENCE_CATEGORY)
             {
                 Console.WriteLine(scienceQuestions.First());
                 scienceQuestions.RemoveFirst();
             }
-            if (CurrentCategory() == "Sports")
+            if (SubjectOfQuestion() == SPORTS_CATEGORY)
             {
                 Console.WriteLine(sportsQuestions.First());
                 sportsQuestions.RemoveFirst();
             }
-            if (CurrentCategory() == "Rock")
+            if (SubjectOfQuestion() == "Rock")
             {
                 Console.WriteLine(rockQuestions.First());
                 rockQuestions.RemoveFirst();
             }
         }
 
 
-        private string CurrentCategory()
+
+        private string SubjectOfQuestion()
         {
-            if (places[currentPlayer] == 0) return "Pop";
-            if (places[currentPlayer] == 4) return "Pop";
-            if (places[currentPlayer] == 8) return "Pop";
-            if (places[currentPlayer] == 1) return "Science";
-            if (places[currentPlayer] == 5) return "Science";
-            if (places[currentPlayer] == 9) return "Science";
-            if (places[currentPlayer] == 2) return "Sports";
-            if (places[currentPlayer] == 6) return "Sports";
-            if (places[currentPlayer] == 10) return "Sports";
+            int positionOnBoard = places[currentPlayer];
+            if ((positionOnBoard == 0) || (positionOnBoard == 4) || (positionOnBoard == 8))  return POP_CATEGORY;            
```

Comment:
Nice - this is much more compact. Have you noticed a pattern between the positions? If you used modulo logic, you could avoid needing the "or" gates.

e.g. if (positonOnBoard % 4 ==0)